src/App.tsx

```diff
@@ -110,7 +110,7 @@ function App() {
 
     if (!isHostProtocolSecure()) {
       showSmallError(`Dashboard is using non-secure protocol! Read more
```

Comment:
remove "Read more" on this line - not necessary to have it both with and without link

Reply:
ooops missed that